boot/boot_test.go

```diff
@@ -643,8 +649,18 @@ version: 5.0
 	// using the UC16 bootmode
 	mockBootGrubenv := filepath.Join(mockBootGrubDir, "grubenv")
 	c.Check(mockBootGrubenv, testutil.FilePresent)
-	c.Check(mockBootGrubenv, testutil.FileContains, "snap_kernel=pc-kernel_5.snap")
-	c.Check(mockBootGrubenv, testutil.FileContains, "snap_core=core20_3.snap")
+	// TODO: is there a better way to check this? we don't want to match if
+	// kernel_status=try, which AFAICT this will still match
+	c.Check(mockBootGrubenv, testutil.FileContains, "kernel_status=")
```

Comment:
Could you use `testutil.FileMatches, "^kernel_status=$")` here?

Reply:
If that's the only content, then can have a stronger check with `c.Check(mockBootGrubenv, testutil.FileEquals, "kernel_status=")`

Reply:
Well the file isn't just this, it has the rest of the grubenv block too, but I made it work with a slight change to @mvo5 's patch:

```go
c.Check(mockBootGrubenv, testutil.FileMatches, `(?m)^kernel_status=$`)
```